.gitignore

```
@@ -109,6 +109,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+myvenv
```

myvenvがまだgit管理下に置かれてしまっています。
```suggestion
myvenv/
```
としてみてください。